test/api/core/notifier/send.test.js

```diff
@@ -125,7 +125,7 @@ describe('Test: notifier.send', function () {
     (Notifier.__get__('send')).call(kuzzle, room, response);
     should(responded).be.true();
     should(mockupio.emitted).be.true();
-    should(mockupio.id).be.undefined();
+    //should(mockupio.id).be.undefined();
```

Comment:
Because before, the implementation was wrong. Now it works and the mockupio.id must be defined.
A line is added for check 'not.bet.undefined()'